src/sentry/static/sentry/app/components/commitRow.jsx

```diff
@@ -7,13 +7,13 @@ import Avatar from 'app/components/avatar';
 import TimeSince from 'app/components/timeSince';
 import CommitLink from 'app/components/commitLink';
 import {t, tct} from 'app/locale';
-
 import {PanelItem} from 'app/components/panels';
 import TextOverflow from 'app/components/textOverflow';
 
 export default class CommitRow extends React.Component {
   static propTypes = {
     commit: PropTypes.object,
+    customAvatar: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
```

Comment:
What if customAvatar just accepted a `node` instead? I think that would actually probably be a little cleaner.